### PR TITLE
fix(component-index): fix horizontal scrolling on component index page

### LINKED ISSUE #1628: Community assets —> Component index: Page should be locked in mobile
In mobile (Chrome) the screen moves all around. Should be locked for left and right horizontal movement.

@andreancardona

### PR DESCRIPTION
**Pull request by aledavila**
Closes #1628 

Fixes the jumping around of the index page on mobile. The issue came from the tags in the component view adding some random extra space. 

#### Changelog

**Changed**

- changed the tags in the footers to absolute positioning.

Check mobile component index page

### CODE REVIEW

**Review by andreancardona — APPROVED** [review]
yay!

File: src/components/ChartIndexPage/chart-index.scss [hunk 1]
```diff
@@ -1,223 +0,0 @@
-//
```
Comment (joshblack) [inline src/components/ChartIndexPage/chart-index.scss:1]: Is this file being used by the Chart Index work that Robin was working on?? 🤔

**Review by jeanservaas — APPROVED** [review]

File: src/components/ChartIndexPage/chart-index.scss (hunk 1 above)
Comment (aledavila) [reply src/components/ChartIndexPage/chart-index.scss:1]: No I think we can remove it with no issue. It was only copied over. But I noticed that all the styles were the same so I removed them and nothing broke.

**Comment by aledavila** [comment]
I removed the file and it doesnt seem to mess anythign up. Mind giving it one more look :) @joshblack

**Review by alisonjoseph — APPROVED** [review]

**Pull request merged**